prow/cmd/deck/main.go

```diff
@@ -659,11 +660,24 @@ func renderSpyglass(sg *spyglass.Spyglass, cfg config.Getter, src string, o opti
 
 	jobHistLink := ""
 	jobPath, err := sg.JobPath(src)
-
 	if err == nil {
 		jobHistLink = path.Join("/job-history", jobPath)
 	}
-	logrus.Infof("job history link: %s", jobHistLink)
+
+	artifactsLink := ""
+	gcswebPrefix := cfg().Deck.Spyglass.GCSBrowserPrefix
+	if gcswebPrefix != "" {
+		runPath, err := sg.RunPath(src)
+		if err == nil {
+			artifactsLink = gcswebPrefix + runPath
+		}
+	}
+
+	prHistLink := ""
+	org, repo, number, err := sg.RunToPR(src)
+	if err == nil {
+		prHistLink = "/pr-history?repo=" + url.QueryEscape(org+"/"+repo) + "&pr=" + strconv.Itoa(number)
```

Comment:
`org` and `repo` should be separate parameters

Reply:
I'm putting this one on the [PR description](https://github.com/kubernetes/test-infra/pull/10933) being misleading.